libvncclient/listen.c

```diff
@@ -30,11 +30,12 @@
 #endif
 #include <sys/types.h>
 #ifdef WIN32
-#define close closesocket
```

Comment:
Well, `close` is used on *nix, `closesocket` on win, `closeSocket` (with "S") is just a fictitious identifier unifying these names in any SO, but it can be replaced by any other identifier, like `Close`. If you declare `#define close closesocket`, you can't compile the library on mingw, and you may get a:
```
...
Making all in libvncclient
make[2]: Entering directory 'C:/dev/git/libvncserver/libvncclient'
  CC       cursor.lo
  CC       listen.lo
listen.c:33:15: error: conflicting types for 'closesocket'
 #define close closesocket
               ^
In file included from listen.c:34:0:
C:/msys32/mingw32/i686-w64-mingw32/include/winsock2.h:975:34: note: previous declaration of 'closesocket' was here
   WINSOCK_API_LINKAGE int WSAAPI closesocket(SOCKET s);
                                  ^~~~~~~~~~~
Makefile:483: recipe for target 'listen.lo' failed
make[2]: *** [listen.lo] Error 1
...
```